dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+const $tasklist = document.querySelector('.task-list');
+
+function removeList(event) {
+
+  console.log('the event target is: ', event.target);
+  console.log('the event target tag name is:', event.target.tagName);
+
+  if (event.target.matches("ul li button")) {
```

Comment:
Works, but way more effort than needed. Why not just do `button`? This is way harder to read